{"name": "test_cant_create_double_courier", "status": "passed", "description": "Проверка невозможности создания курьера с существующим логином. Получаем ошибку 409", "steps": [{"name": "Создание нового курьера", "status": "passed", "parameters": [{"name": "payload", "value": "{'login': 'jkryqrapvz', 'password': 'sflenkvslf', 'firstName': 'uzumxlyhys'}"}], "start": 1715725570143, "stop": 1715725570803}], "start": 1715725570143, "stop": 1715725570807, "uuid": "379d542b-1abf-417c-a933-edce9a2efcc4", "historyId": "779cd0fc13f0cd1872ff621695d12603", "testCaseId": "779cd0fc13f0cd1872ff621695d12603", "fullName": "tests.test_create_courier.TestCreateCourier#test_cant_create_double_courier", "labels": [{"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_create_courier"}, {"name": "subSuite", "value": "TestCreateCourier"}, {"name": "host", "value": "DESKTOP-BQIEDFR"}, {"name": "thread", "value": "15608-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_create_courier"}]}